Homework8.medium/class.js

```diff
@@ -0,0 +1,63 @@
+class shopingCart{
+    constructor() {
+        this.items = [];
+        this.totalPrice = 0
+        
+    }
+
+
+    additem(product, price) {
+        this.items.push(product)
+        this.totalPrice = this.totalPrice + price;
+         
+    }
+    removeItem(product, price) {
+       this.items.shift(product)
+      this.totalPrice = this.totalPrice - price;}
+
+    
+
+     total(items, totalPrice){
```

Comment:
In the `total` method of this class we just want to do a console.log with `this.items.length` (to see how many items we have in our shopping cart) and `this.totalPrice`(to show how much it costs).